### PR TITLE
Add speed and emotion options for Cartesia.

### PR DESCRIPTION
**Pull request by golbin**
I've just added speed and emotion options to Cartesia TTS.

While these are experimental features, they are very useful and could soon be ready for production.

### CODE REVIEW

File: src/pipecat/services/cartesia.py
Comment (aconchillo) [inline src/pipecat/services/cartesia.py:77]: I'm thinking that instead of adding more arguments we could do like `FalImageGenService` or `GladiaSTTService`:

```
class CartesiaTTSService(TTSService):
    class InputParams(BaseModel):
        speed: Optional[str] = None,
        emotion: Optional[List[str]] = None
```

Comment (golbin) [reply src/pipecat/services/cartesia.py:77]: In that case, wouldn’t it be better to include the other options like encoding, sample_rate, and language in the InputParams as well?

Comment (aconchillo) [reply src/pipecat/services/cartesia.py:77]: Yes, I just didn't want to ask you to do more work :sweat_smile:

Comment (golbin) [reply src/pipecat/services/cartesia.py]: No problem, I’ve made a patch based on your advice.

The 'voice_id' parameter is required, so I’ve left it as the default parameter.

Please review and let me know if everything looks good.

Comment (golbin) [reply src/pipecat/services/cartesia.py]: Hmm… maybe we need something like TTSEmotionUpdateFrame and TTSSpeedUpdateFrame? 🤔

File: src/pipecat/services/cartesia.py
Comment (aconchillo) [inline src/pipecat/services/cartesia.py:66]: I'd keep the `model_id` as a main argument as with all the other AI services. The rest looks great!

File: src/pipecat/services/cartesia.py
Comment (aconchillo) [reply src/pipecat/services/cartesia.py]: Not needed for now.

File: src/pipecat/services/cartesia.py
Comment (aconchillo) [reply src/pipecat/services/cartesia.py:66]: There's also a conflict.

**Comment by golbin** [comment]
@aconchillo I fixed conflict and revert "model_id" as a main argument. Please check that.

**Review by aconchillo — APPROVED** [review]

**Comment by aconchillo** [comment]
Looks great! I believe there's one more conflict but I'll merge right away once that's solved. Thank you for your patience!

**Comment by golbin** [comment]
I'm sorry for the delay. The conflict has been resolved.

**Comment by golbin** [comment]
Apply and fix for recent changes. It might be better to refactor CartesiaTTS for inheritance in CartesiaTTSServer/CartesiaHttpService later.

**Comment by williamtran29** [comment]
Can we merge this asap? the speed is too fast atm

**Comment by markbackman** [comment]
@golbin we've made a switch to Ruff as a formatter. This was to resolve formatting issues with Autopep-8. The README has been updated with instructions for setting up Ruff as a linter in your IDE. Can you set up Ruff, fix the linting issues, and push your latest changes? With that either @aconchillo or I can merge it down right away.

**Comment by markbackman** [comment]
Thanks @golbin! 🙌

**Pull request merged**
